README.md

```diff
@@ -48,22 +48,38 @@ Before you attempt to make a contribution please read the [Community Participati
 
 ## Build Instructions
 
+Note: Both Android SDK and NDK are required.
 
 1. Clone or Download the repository:
 
   ```shell
   git clone https://github.com/mozilla-mobile/fenix
   ```
 
-2. Import the project into Android Studio **or** build on the command line:
+2. **Import** the project into Android Studio **or** build on the command line:
 
   ```shell
-  ./gradlew clean app:assembleArmDebug
+  ./gradlew clean app:assembleGeckoBetaDebug
   ```
+  
+  Use app:assembleGeckoNightlyDebug to build with the Gecko Nightly version instead.
 
-3. Make sure to select the correct build variant in Android Studio:
-**armDebug** for ARM
-**x86Debug** for X86
+3. Make sure to select the correct build variant in Android Studio. See the next section.
+
+### Guide to Build Variants
+We have a lot of build variants. Each variant is composed of two flavors. One flavor is the version of Gecko to use and the other describes 
+which app id and settings to use. Here is a description of what each means:
+
+**geckoBeta** (recommended) uses the Beta variant of the Gecko rendering engine, which corresponds to the next version of Gecko which will go to production
+**geckoNightly** uses the Nightly variant of the Gecko rendering engine, which is the version which will arrive after beta and is less stable
+
+**debug** uses debug symbols and debug signing, adds tools like LeakCanary for troubleshooting, and does not strip unused or wasteful code
+**fenixNightly** is a release build with nightly signing which uses the org.mozilla.fenix.nightly app id for nightly releases to Google Play
+**fenixNightlyLegacy** is a release build with release signing which uses the org.mozilla.fenix production app id, which we're trying to phase out
+**fenixBeta** is a release build with beta signing which uses the org.mozilla.fenix.beta app id for beta releases to Google Play
+**fenixProduction** is a release build with release signing which uses the org.mozilla.fenix app id for production relases to Google Play
+**fennecProduction** is an experimental build with release signing which uses the org.mozilla.firefox app id and supports upgrading the older Firefox
```

Comment:
Maybe we should say *not recommended* or mention it causes data loss

Reply:
People who build `fennecProduction` won't have access to Mozilla's signing keys so the would not be able to pave over their Fennec release install.

Reply:
That's right. It will only work on top of their own build of Fennec, not ours.

Reply:
I'll add the data loss warning, since people have been messing with it on Reddit.